charmhelpers/contrib/openstack/files/check_deferred_restarts.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python3
+"""
+Checks for services with deferred service restarts.
+
+This Nagios check will parse /var/lib/policy-rd.d/
+to find any restarts that are currently deferred.
+"""
+
+import argparse
+import glob
+import sys
+import yaml
+
+
+DEFERRED_EVENTS_DIR = '/var/lib/policy-rc.d'
+
+
+def get_deferred_events():
+    """
+    Return a list of deferred events dicts from policy-rc.d files
```

Comment:
I know it's a touch picky, but please could you put the header of the docstring on the same line as the triple quotes (""")

Reply:
Done. I didn't do the same in nrpe.py because the rest of the file has them on different lines. Let me know if you think I should change those as well.

Reply:
No that's fine.  I'm generally in favor of just gradually improving code, but to make those changes as well would probably just confuse the purpose of this PR.  Thanks for make the changes!